platforms-serverless/netlify-cli/test.sh

```diff
@@ -3,9 +3,9 @@
 set -eux
 
 if [ "$PRISMA_CLIENT_ENGINE_TYPE" == "binary" ]; then
-  BINARY_STRING=',"files":["edge","index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]'
+  BINARY_STRING=',"files":["index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]'
 else
-  BINARY_STRING=',"files":["edge","index-browser.js","index.d.ts","index.js","libquery_engine-rhel-openssl-1.0.x.so.node","package.json","runtime","schema.prisma"]'
+  BINARY_STRING=',"files":["index-browser.js","index.d.ts","index.js","libquery_engine-rhel-openssl-1.0.x.so.node","package.json","runtime","schema.prisma"]'
```

Comment:
huh? why is this not present here?

Reply:
The experience is debatable of course, but I would need more time to change the logic to have placeholders just for edge client in custom locations.

Reply:
Oh no, absolutely no need for that - but I was not aware this test uses a custom location. I think that makes the test for the files a bit useless here, or even wrong.

Reply:
On the other hand, the "placeholder" should not have binaries, so I am not sure what is going on.

I am fine with you just making it pass with this change here. But maybe make a note that we need to look into this test here to really figure out what is going on.

Reply:
These here aren't placeholders, they are the actual generated client in a custom location. That's why I said that it does not make sense right now in the code to have placeholders for `edge` in an actual generated client. But we can change that after GA, of course.